### ISSUE TITLE
Custom IP feature should be configurable (enable/disable)

### ISSUE THREAD

**Issue by craSH** [mpolden/echoip#123]
It would be great if the new feature for custom IP interrogation (https://github.com/mpolden/echoip/pull/94) could be enabled or disabled via a command line argument. In some cases, the host may be able to reveal potentially sensitive internal DNS information for private IP addresses depending on deployment/configuration.

**Comment by mpolden** [comment]
Good point. In any case we should deny lookups of private IPs, would that be sufficient?

**Comment by craSH** [comment]
I would welcome that as an option, but I actually have this deployed in my internal network and use it both internally and externally, so I don't want to entirely deny local IP lookups.
